Update activating_regions.mdx (#3765)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/activating_regions.mdx

When you activate a region, BigAnimal prepares the compute and networking resources required to deploy clusters. Note that these added resources can increase your cloud costs. 

> Activation of a region prior to creating or restoring a cluster is required. 

Reviewer: ```suggestion
You must activate a region prior to creating or restoring a cluster. 
```
